charts/factorio/templates/deployment.yaml

```diff
@@ -138,6 +139,9 @@ spec:
         - name: factorio
           containerPort: 34197
           protocol: UDP
+          {{- if .Values.deployment.hostNetwork }}
+          hostPort: {{ .Values.deployment.hostPort }}
+          {{- end }}
```

Comment:
I believe this is not necessary. Host networking means that the pod's network is the same as the bare metal machine. The port being used by the service, whatever it defaults to, is accessible to the host machine. The documentation on all of this is unclear for this property but I think it is ignored? Or at least not needed.

Reply:
Alright, let's remove it then.